src/error/GraphQLError.ts

```diff
@@ -83,7 +83,7 @@ export class GraphQLError extends Error {
     path?: Maybe<ReadonlyArray<string | number>>,
     originalError?: Maybe<(Error & { readonly extensions?: unknown })>,
     extensions?: Maybe<{ [key: string]: unknown }>,
-  ): void {
+  ) {
```

Comment:
Can you please extract this commit into PR against master?

Reply:
@IvanGoncharov created a new PR https://github.com/graphql/graphql-js/pull/2878. I will remove this commit from here.